tests/views/test_general_user_data_view.py

```diff
@@ -193,14 +193,14 @@ def get_course_rec(course_id, course_list):
                 # Test that the course id exists in the data
                 assert get_course_rec(course_enrollment.course_id, rec['courses'])
 
-    def test_get_search(self):
+    @pytest.mark.parametrize('search_term', SEARCH_TERMS)
+    def test_get_search(self, search_term):
         """
         Based on a SEARCH_TERMS data set, we query the endpoint with search
         terms and we compare with the expected results.
         """
         for search_term in SEARCH_TERMS:
```

Comment:
```suggestion
```

Reply:
Thanks @OmarIthawi ! I totally missed this one!